apps/fabric-examples/src/pages/PanelPage/examples/Panel.SmallRight.Example.tsx

```diff
@@ -19,9 +20,50 @@ export class PanelSmallRightExample extends React.Component<any, any> {
           isOpen={ this.state.showPanel }
           type={ PanelType.smallFixedFar }
           onDismiss={ this._closePanel.bind(this) }
-          headerText='Panel - Small, right-aligned, fixed'
+          headerText='Panel - Small, right-aligned, fixed, with footer'
+          onRenderFooterContent={ () => {
+            return (
+              <div>
+                <PrimaryButton
+                  onClick={ this._closePanel.bind(this) }
+                  style={ { 'marginRight': '8px' } } >
+                  Save
+                </PrimaryButton>
+                <DefaultButton
+                  onClick={ this._closePanel.bind(this) }
```

Comment:
`this._closePanel` should be renamed to `_onClosePanel` and should be autobinded